package/MDAnalysis/visualization/streamlines.py

```diff
@@ -226,8 +230,10 @@ def produce_list_centroids_this_frame(list_indices_in_polygon):
     return list(zip(reconstruction_index_list, xy_deltas_to_write.tolist()))
 
 
-def generate_streamlines(topology_file_path, trajectory_file_path, grid_spacing, MDA_selection, start_frame,
-                         end_frame, xmin, xmax, ymin, ymax, maximum_delta_magnitude, num_cores='maximum'):
+def generate_streamlines(topology_file_path: str, trajectory_file_path: str, grid_spacing: float, MDA_selection: str,
+                         start_frame: int, end_frame: int,
+                         xmin: float, xmax: float, ymin: float, ymax: float, maximum_delta_magnitude: float,
+                         num_cores: int = 'maximum') -> Tuple[Sequence[float], Sequence[float], float, float]:
```

Comment:
Here too, the annotations for the return values need to be specific. The sequences seem to be arrays.